reports/migrations/0006_auto_20210325_2053.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.19 on 2021-03-25 20:53
```

Comment:
Do we really need this migration generated here? I think we need to generate this in `ata_portal` instead

Reply:
True. I guess portal is the right place - we'll generate migrations in the other PR after this one is updated as a dependency in `requirements.txt`?

Reply:
All the migrations required to use this as a third party library are included here. This would be one of those.

Reply:
Thats what I thought @jseadragon.

@vimpadawan yes, I see that, I think its more correct to generate them in the `ata_portal`. But we can keep this here as we have done with the others.